Shared/Themes/MergedResourceDictionaries.xaml

```diff
@@ -0,0 +1 @@
+<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" />
```

Comment:
this file should not be here (.gitignore) and Generic.xaml can be deleted

Reply:
and since this file is empty, it means it does not work.

Reply:
This was pushed by mistake, since we now only use XamlMerge for Uno, the Shared project shouldn't have those